script/changelog/parser.ts

```diff
@@ -0,0 +1,121 @@
+import * as Path from 'path'
+import * as Fs from 'fs'
+import { gt } from 'semver'
+
+import { fetchPR, IAPIPR } from './api'
+
+const PlaceholderChangeType = '???'
+const OfficialOwner = 'desktop'
+
+interface IParsedCommit {
```

Comment:
Will this need to be updated to handle co-authors

Reply:
Let's save this discussion for #3949 or later -  this was intended to be extracted without changing the behaviour